source/_docs/pantheon-yml.md

```diff
@@ -111,6 +72,25 @@ Add `drush_version` to the top level of the `pantheon.yml` file to configure the
 drush_version: 8
 ```
 For more information, see [Managing Drush Versions on Pantheon](/docs/drush-versions).
+
+
+### Custom Upstreams and pantheon.upstream.yml
```

Comment:
Suggest h2 instead of h3 there, so this content isn't nested within the advanced site configuration section.